### PR TITLE
Removes the access restriction from wall mounted nano meds

### PR DESCRIPTION
**Pull request by MrsTonedOne**
This doesn't make sense to have, the full ones should, but not the smaller ones.

### CODE REVIEW

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -764,8 +764,7 @@ IF YOU MODIFY THE PRODUCTS LIST OF A MACHINE, MAKE SURE TO UPDATE ITS RESUPPLY C
 	desc = "Wall-mounted Medical Equipment dispenser."
 	icon_state = "wallmed"
 	icon_deny = "wallmed-deny"
-	req_access_txt = "5"
```
Comment (neersighted) [inline code/game/machinery/vending.dm]: Why was any non-mapedited item using req_access_text and not req_access... I cry

**Comment by Zilenan91** [comment]
:+1:  :x:  :100:

**Comment by Zilenan91** [comment]
Any chance of you putting these in some of the departments on box in case of emergencies?

File: code/game/machinery/vending.dm [hunk 2]
```diff
@@ -764,8 +764,7 @@ IF YOU MODIFY THE PRODUCTS LIST OF A MACHINE, MAKE SURE TO UPDATE ITS RESUPPLY C
 	desc = "Wall-mounted Medical Equipment dispenser."
 	icon_state = "wallmed"
 	icon_deny = "wallmed-deny"
-	req_access_txt = "5"
-	density = 0 //It is wall-mounted, and thus, not dense. --Superxpdude
+	density = 0
 	products = list(/obj/item/weapon/reagent_containers/syringe = 3,/obj/item/weapon/reagent_containers/pill/patch/styptic = 5,
```
Comment (Jalleo) [inline code/game/machinery/vending.dm]: Its a bit redundant for it to have syringes and no liquid containers. Also those pills should really be pill bottles or they run out before one person uses them.

Comment (tkdrg) [inline code/game/machinery/vending.dm]: that's pretty tangential to this PR, you're free to make a new one changing those though

File: code/game/machinery/vending.dm (hunk 1 above)
Comment (Giacom) [inline code/game/machinery/vending.dm]: Because no one could figure out how to enter lists into the map vars.

**Pull request merged**